src/ijson/__init__.py

```diff
@@ -49,11 +49,13 @@ def _default_backend():
 parse_coro = backend.parse_coro
 items = backend.items
 items_coro = backend.items_coro
+prefixed_items = backend.prefixed_items
```

Comment:
This is a good place as any to discuss the name. I'm not convinced by "prefixed items" doesn't convey the fact that now you can specify multiple prefixes. I originally thought of "multi_items" or something along those lines, to make it clearer that there was no restriction on the number of prefixes.